### DISCUSSION TITLE
Co-maintainers

### DISCUSSION THREAD

**Discussion by dhui** [golang-migrate/migrate#740] [General]
Thread to discuss adding new co-maintainers.

CC: @Fontinalis

**Comment by dhui** [comment]
@pankona
RE: larger contribution to golang-migrate
Do any [open issues](https://github.com/golang-migrate/migrate/issues) seem interesting to you?
I think adding new features, fixing bugs, revamping docs, and/or helping others debug their issues would count as a "larger contribution"

**Reply by pankona** [reply]
@dhui 

Thank you for picking me up on the discussion!
Sure, I'll check the open issues and I'll try to make contributions.

**Comment by ldej** [comment]
This project appears to be quite popular. Of all the golang-specific database migration tools, this one has the most stars. In the past years I have used migrate with postgres, spanner and now immudb (https://github.com/golang-migrate/migrate/pull/749). 

Please don't let this project die. What would be needed to get the spirit going again?
